Userland/Services/SystemServer/Service.cpp

```diff
@@ -398,3 +399,16 @@ bool Service::is_enabled() const
     extern String g_system_mode;
     return m_system_modes.contains_slow(g_system_mode);
 }
+
+void Service::determine_account(int fd)
+{
+    // FIXME: Should propagate errors
```

Comment:
Just use TRY+ErrorOr here and ignore it at the call site, much easier than ignoring failures in multiple places and fixing that up later.